courses/machine_learning/deepdive2/image_classification/solutions/3_tf_hub_transfer_learning.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<p>Can you also mention that transfer learning can be achieved using Keras Applications?</p><p> </p><p><a href="https://keras.io/applications/" target="_blank">https://keras.io/applications/</a></p><p> </p><p>But I would say that the benefit of TF-Hub is that it has a wider variety of models, including the state of the art models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Yeah, I had it in an earlier version of the lab, but wasn't sure where to put it. Since the bonus challenge is Keras based, I added it there. Thoughts?